ns/vocab.csv

```diff
@@ -1,72 +1,70 @@
+numericFacet,rdf:Property,,xsFacet,,,,,,,,Abstract property of numeric facets on a NodeConstraint.
+stringFacet,rdf:Property,,xsFacet,,,,,,,,An abstract property of string facets on a NodeConstraint.
+Annotation,rdfs:Class,Annotation,,,,,,,,,Annotations provide a format-independent way to provide additional information about elements in a schema. 
+EachOf,rdfs:Class,Each Of,TripleExpression,,,,,,,,"A TripleExpression composed of one or more sub-expressions, all of which must match."
+NodeConstraint,rdfs:Class,Node Constraint,ShapeExpression,,,,,,,,A constraint on the type or value of an RDF Node.
+NodeKind,rdfs:Class,Node Kind,,,,,,,,,The set of kinds of RDF Nodes.
+OneOf,rdfs:Class,One Of,TripleExpression,,,,,,,,"A TripleExpression composed of one or more sub-expressions, one of which must match."
+Schema,rdfs:Class,Schema,,,,,,,,,"A Schema contains the set of shapes, used for matching a focus node."
+SemAct,rdfs:Class,Semantic Actions,,,,,,,,,"A list of Semantic Actions that serve as an extension point for Shape Expressions. They appear in lists in Schema's startActs and Shape, OneOf, EachOf and TripleConstraint's semActs."
+Shape,rdfs:Class,Shape Or,ShapeExpression,,,,,,,,A shapes schema is captured in a Schema object where shapes is a mapping from shape label to shape expression.
+ShapeAnd,rdfs:Class,Shape And,ShapeExpression,,,,,,,,"A ShapeExpression composed of one or more sub-expressions, all of which must match."
+ShapeExpression,rdfs:Class,Shape Expression,,,,,,,,,The abstract class of Shape Expressions.
+ShapeExternal,rdfs:Class,Shape External,ShapeExpression,,,,,,,,A reference to a shape defined in some external Schema.
+ShapeNot,rdfs:Class,Shape Not,ShapeExpression,,,,,,,,A ShapeNot is satisfied when it’s included ShapeExpression is not satisfied.
+ShapeOr,rdfs:Class,Shape Or,ShapeExpression,,,,,,,,"A ShapeExpression composed of one or more sub-expressions, one of which must match."
+Stem,rdfs:Class,Stem,,,,,,,,,An IRI prefix used for matching IRIs.
+StemRange,rdfs:Class,StemRange,,,,,,,,,"An IRI prefix (or wildcard) along with a set of excluded values, used for node matching."
+TripleConstraint,rdfs:Class,Triple Constraint,TripleExpression,,,,,,,,A constraint on a triple having a specific predicate and optionally a shape expression used for matching values.
+TripleExpression,rdfs:Class,Triple Expression,,,,,,,,,The abstract class of Triple Expressions.
+Wildcard,rdfs:Class,Wildcard,,,,,,,,,"Indicates that a stem is a Wildcard, rather than a URI prefix."
+annotations,term,,annotation,,,@id,,,,,A synonym for the annotation property.
+exclusions,term,,exclusion,,,@id,,,,,A synonym for the exclusion property.
+id,term,,@id,,,,,,,,A synonym for @id.
+language,term,,@language,,,,,,,,A synonym for @language.
+type,term,,@type,,,,,,,,A synonym for @type.
+uri,term,,@id,,,,,,,,A synonym for @id.
+value,term,,@value,,,,,,,,A synonym for @value.
```

Comment:
I can't tell what changes here due to problems Git has with CSVs, can you describe?